compiler/rustc_middle/src/ty/vtable.rs

```diff
@@ -10,10 +10,8 @@ use crate::ty::{self, Instance, PolyTraitRef, Ty, TyCtxt};
 pub enum VtblEntry<'tcx> {
     /// destructor of this type (used in vtable header)
     MetadataDropInPlace,
-    /// layout size of this type (used in vtable header)
-    MetadataSize,
-    /// layout align of this type (used in vtable header)
-    MetadataAlign,
+    /// size and align of this type (used in vtable header)
```

Comment:
(Only relevant if this proceeds to an actual proposal.)

Please explain how the encoding works.